### PR TITLE
feat: update mobile optimization

### PR DESCRIPTION
**Pull request by Jahanzeb9999**
This change is [Reviewable](https://reviewable.io/reviews/CoreumFoundation/docs-next/70)

### CODE REVIEW

File: .github/workflows/build-and-deploy.yml [hunk 1]
```diff
@@ -76,16 +76,7 @@ jobs:
...
-          echo "Image: $IMAGE"
-          echo "Tag: ${{ matrix.TAG }}"
-
       - name: Build
-        env:
-          NEXT_PUBLIC_ALGOLIA_APP_ID: ${{ secrets.NEXT_PUBLIC_ALGOLIA_APP_ID }}
-          NEXT_PUBLIC_ALGOLIA_API_KEY: ${{ secrets.NEXT_PUBLIC_ALGOLIA_API_KEY }}
         run: |-
           docker build \
             --build-arg NEXT_PUBLIC_ALGOLIA_APP_ID=${NEXT_PUBLIC_ALGOLIA_APP_ID} \
```
Comment (alinetskyi) [inline .github/workflows/build-and-deploy.yml:82]: please use here `${{ secrets.NEXT_PUBLIC_ALGOLIA_APP_ID }}` instead of `${NEXT_PUBLIC_ALGOLIA_APP_ID}`, same for the other env

File: next.config.mjs [hunk 2]
```diff
@@ -47,13 +47,11 @@ const nextConfig = {
   reactStrictMode: true,
   output: "standalone",
   staticPageGenerationTimeout: 1000,
-  reactStrictMode: true,
-  env: {
+  publicRuntimeConfig: {
```
Comment (alinetskyi) [inline next.config.mjs:50]: Is this change necessary? It worked fine before... Have you tested it? 
Also, it looks like a deprecated feature
Screenshot 2024-09-04 at 16 35 51

File: yarn.lock [hunk 3]
```diff
@@ -93,11 +93,6 @@
     "@algolia/requester-common" "4.24.0"
     "@algolia/transporter" "4.24.0"
```
Comment (alinetskyi) [inline yarn.lock:94]: There are conflicts with master, please fix them

**Review by alinetskyi — CHANGES_REQUESTED** [review]

File: .github/workflows/build-and-deploy.yml (hunk 1 above)
Comment (Jahanzeb9999) [reply .github/workflows/build-and-deploy.yml]: fixed

File: next.config.mjs (hunk 2 above)
Comment (Jahanzeb9999) [reply next.config.mjs]: Fixed

**Review by alinetskyi — APPROVED** [review]
